package.json

```diff
@@ -25,21 +25,21 @@
     "repositoryUrl": "https://github.com/cerner/cucumber-forge-report-generator.git"
   },
   "dependencies": {
-    "@types/i18next": "^12.1.0",
-    "@types/line-reader": "0.0.28",
     "handlebars": "^4.1.2",
-    "i18next": "^1.10.6",
-    "line-reader": "^0.2.4",
-    "moment": "^2.22.2"
+    "i18next": "^17.0.11",
+    "i18next-node-fs-backend": "^2.1.3",
+    "line-reader": "^0.4.0",
+    "moment": "^2.24.0",
+    "npm-check-updates": "^3.1.21"
   },
   "devDependencies": {
-    "chai": "^4.1.2",
+    "chai": "^4.2.0",
     "cucumber": "^5.1.0",
-    "eslint": "^5.15.3",
-    "eslint-config-airbnb": "^17.1.0",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-jsx-a11y": "^6.1.1",
-    "eslint-plugin-react": "^7.11.1",
-    "jsdom": "^12.0.0"
+    "eslint": "^6.1.0",
+    "eslint-config-airbnb": "^18.0.0",
```

Comment:
could you try swapping `eslint-config-airbnb` to `eslint-config-airbnb-base`?

`-base` has fewer dependencies. Easiest way to swap it is....
```sh
npm remove -D eslint-config-airbnb
npx install-peerdeps --dev eslint-config-airbnb
```

this'll make sure we're using the miniaml needed, specific to what we need, instead of the additional react and accessibility stuff.